torch/fx/passes/infra/pass_manager.py

```diff
@@ -133,7 +155,7 @@ class PassManager:
             after each pass
     """
 
-    passes: List[Callable[[nn.Module], nn.Module]] = []
+    passes: List[Callable[[nn.Module], PassResult]] = []
```

Comment:
You mentioned some passes works with nn.Module, could you please share a code pointer for such case. 

I am thinking if we should constrain the PassBase to only work with GraphModule.